keyboards/handwired/at101ish/keymaps/osdetect/rules.mk

```diff
@@ -0,0 +1,2 @@
+OS_DETECTION_ENABLE = yes
+DEFERRED_EXEC_ENABLE = yes
```

Comment:
This doesn't appear to be used?

Reply:
OS detection is in the osdetect keymap. Not entirely sure DEFERRED_EXEC_ENABLE is needed any more. One documentation page said it was needed, but it used deferred exec as a callback for the os detection. Found the process_detected_host_os_kb/user function later on.

Reply:
Yeah, only asking about the `DEFERRED_EXEC_ENABLE`. It can be removed in that case.

Reply:
For reference, if a feature needs another feature, it will explicitly enable it.  So if you're not  sure, then you don't need it.  Worst case, you'll get compilation errors, and that will confirm things.